docs/api.rst

```diff
@@ -157,4 +158,4 @@ Simulation
     :toctree: _api
     :template: function.rst
 
-    simulation.simulate_diffusive_tracks
\ No newline at end of file
+    simulation.simulate_diffusive_tracks
```

Comment:
Is GitHub tripping?

Reply:
I added a newline at the end of the file 😅